test/Actions.test.ts

```diff
@@ -130,4 +130,109 @@ contract('Actions', ([owner, random]) => {
       assert.equal(depositArgs.index, new BN(index))
     })
   })
+  describe('Parse Withdraw Arguments', () => {
+    it('should not be able to parse a non withdraw action', async () => {
+      const actionType = ActionType.OpenVault
+      const asset = ZERO_ADDR
+      const vaultId = '0'
+      const amount = '10'
+      const index = '0'
+      const bytesArgs = ZERO_ADDR
+
+      const data = {
+        actionType: actionType,
+        owner: owner,
+        sender: owner,
+        asset: asset,
+        vaultId: vaultId,
+        amount: amount,
+        index: index,
+        data: bytesArgs,
+      }
+
+      await expectRevert(
+        actionTester.testParseWithdrawAction(data),
+        'Actions: can only parse arguments for withdraw actions',
+      )
+    })
+    it('should not be able to parse an invalid sender address', async () => {
+      const actionType = ActionType.WithdrawCollateral
+      const asset = ZERO_ADDR
+      const vaultId = '0'
+      const amount = '10'
+      const index = '0'
+      const bytesArgs = ZERO_ADDR
+
+      const data = {
+        actionType: actionType,
+        owner: ZERO_ADDR,
+        sender: owner,
+        asset: asset,
+        vaultId: vaultId,
+        amount: amount,
+        index: index,
+        data: bytesArgs,
+      }
+
+      await expectRevert(actionTester.testParseWithdrawAction(data), 'Actions: cannot withdraw to an invalid account')
```

Comment:
This is actually testing an invalid `owner` address. 
Should have both `args.owner` and `args.sender` checks in the contract (As suggested above) and have 2 separate tests here.

Reply:
hmm what do you mean?

Reply:
Should the check be happening here or in the controller?

Reply:
You have 2 require checks right?
require(_args.owner != address(0), "Actions: cannot withdraw from an invalid vault");
require(_args.sender != address(0), "Actions: cannot withdraw to an invalid account");

Just saying you need test for both of them